chainer/functions/pooling/max_pooling_nd.py

```diff
@@ -21,6 +21,13 @@ class MaxPoolingND(pooling_nd._PoolingND):
 
     """Max pooling over a set of N-dimensional planes."""
 
+    def __init__(self, ndim, ksize, stride=None, pad=0, cover_all=True,
+                 use_cudnn=True):
+        experimental('chainer.functions.pooling.MaxPoolingND')
```

Comment:
Could you use `util.experimental` as direct import of modules are prohibited by pep8. (Should we change the document of `chainer.util.experimental as well?`)